cmd/rekor-server/app/serve.go

```diff
@@ -74,6 +75,9 @@ var serveCmd = &cobra.Command{
 		}
 		log.Logger.Infof("starting rekor-server @ %v", viStr)
 
+		reg := prometheus.NewRegistry()
+		reg.MustRegister(api.NewVersionCollector("rekor", vi))
```

Comment:
I think the promauto constructors register automatically, so this can be removed.

https://pkg.go.dev/github.com/prometheus/client_golang/prometheus/promauto

Reply:
true, thanks for the reminder